Portal/src/Datahub.Portal/Components/Markdown/DHMarkdownEditor.razor

```diff
@@ -0,0 +1,49 @@
+@*Markdown.razor*@
+
+<MarkdownEditor Value="@MarkdownValue" ValueChanged="OnMarkdownValueChanged" ValueHTML="@MarkdownHtml" ValueHTMLChanged="OnMarkdownValueHTMLChanged" @ref="_elementRef"/>
+
+@code {
+    [Parameter]
+    public string InitialValue { get; set; }
+
+    [Parameter]
+    public string MarkdownValue { get; set; }
+
+    [Parameter]
+    public string MarkdownHtml { get; set; }
+
+    [Parameter]
+    public EventCallback<string> ValueChanged { get; set; }
+
+    [Parameter]
+    public EventCallback<string> HtmlChanged { get; set; }
```

Comment:
Would there be a way we could simplify the parameters? I'm not quite familiar with the new library but it would be ideal if we only had the content and content changed params

Reply:
Cleaned up with newest commit